packages/accordion/README.md

```diff
@@ -0,0 +1,37 @@
+# @littlespoon/accordion
+
+[![NPM version](https://img.shields.io/npm/v/@littlespoon/accordion.svg)](https://www.npmjs.com/package/@littlespoon/accordion)
+
+## Install
+
+[Yarn](https://yarnpkg.com/package/@littlespoon/accordion):
+
+```sh
+yarn add @littlespoon/accordion
+```
+
+[npm](https://www.npmjs.com/package/@littlespoon/accordion):
+
+```sh
+npm install @littlespoon/accordion --save
+```
+
+[UNPKG](https://unpkg.com/browse/@littlespoon/accordion/)
+
+## Usage
+
+```ts
+import Accordion from '@littlespoon/accordion'
+```
+
+Render Accordion:
+
+```tsx
+<Accordion />
+```
+
+Render Accordion with props:
+
+```tsx
+<Accordion items={items} />
```

Comment:
@fjallkraken shared this link as a reference.
https://github.com/little-spoon-dev/little-spoon-frontend/blob/main/app/components/utility/AccordionV2/index.js
I think the current data-based architecture is better to replace the existing AccodionV2 component.
@remarkablemark , any thoughts?

Reply:
I'll trust @fjallkraken's judgement here but if we're going to build it right, we should follow React best practices

Reply:
I agree with @remarkablemark here. This approach is perfectly fine, but since we want this to be more generic and customizable, it's probably better to take the component-based approach.

The data approach made a little more sense in the FE because we could always change the styling there if we wanted. We could also use this component to replace the internals of `AccordionV2` that uses the data approach.

What do you think @ryanjdiercks?

Reply:
Yeah, my original purpose was to replace the old `AccordionV2` without any big changes.
I am okay to take the component-based approach but it would take some to refactor it again.
@remarkablemark

Reply:
Yes let's focus on what's best for long term since this is design-system, not internal

Reply:
@remarkablemark , @fjallkraken , I fully refactored the component based on component based architecture.
Please review it again and let me know your thoughts.